tests/test_bgp_af.rb

```diff
@@ -314,65 +371,59 @@ def test_additional_paths_selection
     #
     # Set and verify
     #
+    if platform == :ios_xr
+      config_ios_xr_dependencies(asn)
+      config('route-policy drop_all', '#drop_all', 'end-policy')
```

Comment:
Out of my area of expertise, can you suggest an alternative?  It looks like XR needs some additional setup before configuring the route-policy for additional-paths.  I think this is meant to be a mock policy?  Would it be better to setup a 'realistic' policy instead?